source/ShopPanel.cpp

```diff
@@ -388,6 +388,15 @@ bool ShopPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, boo
 		return SetScrollToTop();
 	else if(key == SDLK_END)
 		return SetScrollToBottom();
+	else if(key == 'k' || (key == 'p' && (mod & KMOD_SHIFT)))
+	{
+		const Ship *flagship = player.Flagship();
+		bool anyEscortUnparked = any_of(playerShips.begin(), playerShips.end(),
+			[&](const Ship *ship){ return ship != flagship && !ship->IsParked() && !ship->IsDisabled(); });
+		for(const Ship *ship : playerShips)
+			if(ship != flagship && !ship->IsDisabled())
```

Comment:
Doesn't ParkShip have its own check for disabled ships?
```suggestion
			if(ship != flagship)
```

Reply:
Looks like it does. I had not noticed. It also loops through all ships until it finds the one you passed in?!

Reply:
The flagship check could probably be added to ParkShip too, though not in this PR.